beacontools/scanner.py

```diff
@@ -168,6 +168,7 @@ def process_packet(self, pkt):
         if not ( \
             ((self.mode & ScannerMode.MODE_IBEACON) and (pkt[19:23] == b"\x4c\x00\x02\x15")) or \
             ((self.mode & ScannerMode.MODE_EDDYSTONE) and (pkt[19:21] == b"\xaa\xfe")) or \
+            ((self.mode & ScannerMode.MODE_ESTIMOTE) and (pkt[23:25] == b"\x5d\x01")) or \
```

Comment:
I wasn't sure if reusing the same device filter and scan mode setting would be acceptable or not.